Modules/Relationships/src/Relationships.Domain/Aggregates/Relationships/Relationship.CreateTests.cs

```diff
@@ -82,28 +82,12 @@ public void Cannot_create_Relationship_to_self()
         acting.Should().Throw<DomainException>().WithError("error.platform.validation.relationshipRequest.cannotSendRelationshipRequestToYourself");
     }
 
-    [Fact]
-    public void There_can_only_be_one_active_relationship_between_two_identities()
-    {
-        // Arrange
-        var existingRelationships = new List<Relationship>
-        {
-            CreateActiveRelationship()
-        };
-
-        // Act
-        var acting = () => new Relationship(RELATIONSHIP_TEMPLATE_OF_1, IDENTITY_2, DEVICE_2, null, existingRelationships);
-
-        // Assert
-        acting.Should().Throw<DomainException>().WithError("error.platform.validation.relationshipRequest.relationshipToTargetAlreadyExists");
-    }
-
     [Theory]
     [InlineData(RelationshipStatus.Pending)]
     [InlineData(RelationshipStatus.Active)]
     [InlineData(RelationshipStatus.Terminated)]
     [InlineData(RelationshipStatus.DeletionProposed)]
-    public void There_can_only_be_one_active_relationship_between_two_identities2(RelationshipStatus relationshipStatus)
+    public void New_relationship_cannot_be_created_when_there_is_an_existing_one_in_status_x(RelationshipStatus relationshipStatus)
```

Comment:
I'm still not sure why we need another test for `Terminated` status at the end? Doesn't this cover everything?

Reply:
We don't. I'm just stupid.